src/Common/src/Common.Security/ConfigureCertificateOptions.cs

```diff
@@ -40,7 +40,7 @@ public void Configure(string name, CertificateOptions options)
             return;
         }
 
-        options.Certificate = new X509Certificate2(certPath, string.Empty, X509KeyStorageFlags.EphemeralKeySet);
+        options.Certificate = new X509Certificate2(certPath);
```

Comment:
By Default this is not EphemeralKeySet ([docs](https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509keystorageflags?view=net-7.0)). I also found this [blog](https://www.pkisolutions.com/handling-x509keystorageflags-in-applications/).  I am also confused as to why this would fix the issue.